.golangci.yml

```diff
@@ -66,9 +58,5 @@ linters:
 issues:
   exclude:
     - "composite literal uses unkeyed fields" # govet
-#   exclude-use-default: true
-#   max-per-linter: 0
-#   max-same: 0
-#   new: false
-#   new-from-rev: ""
-#   new-from-patch: ""
+    - "G304: Potential file inclusion via variable" # gosec
+    - "G104: Errors unhandled." #gosec
```

Comment:
I disabled it because it reports errors when an error is handled but discarded, for example `_ := someFn()` reports an error.